### ISSUE TITLE
Error on 0.6.0, after update

### ISSUE THREAD

**Issue by Romanior** [RuslanZavacky/ember-cli-ifa#27]
```javascript
ember.debug.js:19750 TypeError: Cannot read property 'getAttribute' of null
    at Object.initialize (asset-map.js:16)
    at ember.debug.js:5556
    at ember.debug.js:5585
    at Object.visit [as default] (ember.debug.js:54608)
    at DAG.topsort (ember.debug.js:2578)
    at Class._runInitializer (ember.debug.js:5584)
    at Class.runInitializers (ember.debug.js:5545)
    at Class._bootSync (ember.debug.js:4771)
    at Class.domReady (ember.debug.js:4660)
    at Backburner.run (ember.debug.js:294)
```

```
➜  git:(greenkeeper/ember-cli-ifa-0.6.0) ✗ ember -v
ember-cli: 2.15.0
node: 8.5.0
os: darwin x64
```

`package.js`
```json
  "devDependencies": {
    "babel-preset-env": "1.6.0",
    "babelify": "7.3.0",
    "blueimp-md5": "^2.10.0",
    "body-parser": "^1.14.1",
    "braintree": "2.3.0",
    "broccoli-asset-rev": "^2.2.0",
    "chai-jquery": "^2.0.0",
    "chalk": "^2.0.0",
    "color": "^2.0.0",
    "connect-restreamer": "1.0.3",
    "d3-array": "1.2.1",
    "d3-collection": "1.0.4",
    "d3-scale": "1.0.4",
    "dotenv": "^4.0.0",
    "ember-ajax": "^3.0.0",
    "ember-basic-dropdown": "0.33.6",
    "ember-browserify": "1.2.0",
    "ember-buffered-proxy": "^0.7.0",
    "ember-changeset": "1.3.0",
    "ember-cli": "2.15.0",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-autoprefixer": "0.8.1",
    "ember-cli-babel": "^6.7.1",
    "ember-cli-bugsnag": "~1.3.0",
    "ember-cli-chai": "0.4.3",
    "ember-cli-chartist": "^1.0.0",
    "ember-cli-clipboard": "0.8.0",
    "ember-cli-content-security-policy": "1.0.0",
    "ember-cli-dependency-checker": "^2.0.1",
    "ember-cli-deploy": "1.0.0",
    "ember-cli-deploy-build": "1.0.0",
    "ember-cli-deploy-display-revisions": "1.0.0",
    "ember-cli-deploy-gzip": "1.0.0",
    "ember-cli-deploy-notifications": "0.2.0",
    "ember-cli-deploy-revision-data": "1.0.0",
    "ember-cli-deploy-s3": "1.1.0",
    "ember-cli-deploy-s3-index": "1.0.0",
    "ember-cli-deprecation-workflow": "0.2.3",
    "ember-cli-eslint": "4.2.1",
    "ember-cli-htmlbars": "^2.0.3",
    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
    "ember-cli-ifa": "^0.6.0",
    "ember-cli-inject-live-reload": "^1.4.0",
    "ember-cli-is-component": "0.5.0",
    "ember-cli-less": "1.5.5",
    "ember-cli-mirage": "0.3.4",
    "ember-cli-mocha": "0.14.4",
    "ember-cli-moment-shim": "3.5.0",
    "ember-cli-page-object": "0.9.0",
    "ember-cli-release": "0.2.9",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-test-loader": "2.2.0",
    "ember-cli-uglify": "^2.0.0",
    "ember-composable-helpers": "^2.0.1",
    "ember-concurrency": "0.8.7",
    "ember-concurrency-test-waiter": "0.2.0",
    "ember-cookies": "0.0.13",
    "ember-data": "2.7.0",
    "ember-disable-proxy-controllers": "^1.0.1",
    "ember-exam": "0.8.0",
    "ember-export-application-global": "^2.0.0",
    "ember-font-awesome": "4.0.0-alpha.10",
    "ember-group-by": "0.0.4",
    "ember-i18n": "5.0.2",
    "ember-in-viewport": "2.1.1",
    "ember-inflector": "^2.0.0",
    "ember-load-initializers": "^1.0.0",
    "ember-moment": "7.4.1",
    "ember-notify": "5.2.0",
    "ember-power-calendar": "0.5.0",
    "ember-power-select-with-create": "0.4.3",
    "ember-radio-button": "1.1.1",
    "ember-resolver": "^4.1.0",
    "ember-route-action-helper": "2.0.6",
    "ember-route-history": "0.1.3",
    "ember-simple-auth": "1.4.0",
    "ember-tether": "1.0.0-beta.0",
    "ember-truth-helpers": "1.3.0",
    "ember-windoc": "0.2.0",
    "eslint-plugin-ember-best-practices": "^1.0.0",
    "faker": "3.0.1",
    "flatten-hash": "^0.0.4",
    "git-repo-info": "1.4.1",
    "glob": "^7.1.1",
    "loader.js": "^4.0.1",
    "lodash": "^3.9.3",
    "moment": "2.18.1",
    "moment-timezone": "0.5.13",
    "onesky": "^0.1.6",
    "rsvp": "^4.6.1",
    "yargs": "9.0.1"
  },
  "ember-addon": {
    "paths": [
      "lib/google-fonts",
      "lib/intercom",
      "lib/locales-cleanup"
    ]
  },
  "greenkeeper": {
    "ignore": [
      "ember-cli",
      "ember-cli-page-object",
      "ember-data",
      "faker",
      "lodash"
    ]
  },
  "dependencies": {}
}

```

`bower.js`
```json
{
  "name": "",
  "dependencies": {
    "chartist-plugin-tooltip": "^0.0.15",
    "chartist": "~0.9.8",
    "ember-cli-moment-shim": "0.0.3",
    "ember-cli-shims": "0.1.1",
    "ember": "2.7.0",
    "jquery": "^2.1.4",
    "lodash": "~3.7.0",
    "moment-duration-format": "^1.3.0",
    "moment-range": "1.0.9",
    "nouislider": "~7.0.10",
    "platform.js": "platform#~1.3.0",
    "sinonjs": "1.14.1",
    "tb-factory": "1.2.1"
  }
}

```

Any clues ?

**Comment by RuslanZavacky** [comment]
@Romanior My bad, it is definitely the latest release. Do you use IFA inline or as a script?

**Comment by RuslanZavacky** [comment]
Hm, did some checking. It should work for both cases. Do you have 

```
{{content-for "head"}}
```

in your index.html?

**Comment by RuslanZavacky** [comment]
I've released version 0.6.1, that should fix it for you @Romanior :)

**Comment by Romanior** [comment]
thanks Ruslan, it is working now!
